package.json

```diff
@@ -15,11 +15,12 @@
     "build": "tsc --project tsconfig.build.json",
     "build:clean": "rimraf dist && yarn build",
     "build:docs": "typedoc",
-    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:dependencies",
+    "lint": "yarn lint:eslint && yarn lint:prettier --check && yarn lint:dependencies && yarn lint:changelog",
+    "lint:changelog": "auto-changelog validate",
     "lint:dependencies": "depcheck",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:dependencies",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:prettier --write && yarn lint:dependencies",
```

Comment:
Should this include `lint:changelog` as well? There is no auto-fixer, but the same is true of `lint:dependencies`.

Reply:
I guess it doesn't hurt. Added in [f3bd4e8](https://github.com/MetaMask/metamask-module-template/pull/174/commits/f3bd4e8578f1bfade0f14c896b521d9ab5bc3594).